src/app/file-explorer.js

```diff
@@ -251,9 +239,9 @@ function fileExplorer (appAPI, files) {
     element = el
   }
 
-  element.events = events
-  element.api = api
-  return element
+  api.events = events
+  container.appendChild(element)
+  return api
```

Comment:
I think it would be more consistent if this function would be used as constructor and then returns an object that has a property `api`.